Lib/nturl2path.py

```diff
@@ -24,22 +24,14 @@ def url2pathname(url):
             # convert this to \\host\path\on\remote\host
             # (notice halving of slashes at the start of the path)
             url = url[2:]
-        components = url.split('/')
-        # make sure not to convert quoted slashes :-)
```

Comment:
This comment implies that quoted forward slashes shouldn't be converted, so a URL like `file:///c:/foo%2fbar` should result in `c:\foo/bar`. But I can't see any reason for this, given forward and backward slashes are treated identically by most Windows programs, even going back to the first versions of Windows. Python path operations almost always treat them as interchangeable, e.g. plenty of code does `p.replace(os.path.altsep, os.path.sep)`

Reply:
Unfortunately, we need to consider how this affects people who parse/convert and then do comparisons. There are a bunch of "security" researchers going around finding ways for blocklists to break[^1] and forcing us to announce vulnerabilities.

[^1]: Spoiler: blocklists are always broken. Use a safelist.

Reply:
Fair point! I've reverted the change and added a test.